prisma/schema.prisma

```diff
@@ -9,171 +8,277 @@ datasource db {
+  name      String
   createdAt DateTime @default(now())
   updatedAt DateTime @default(now())
 
-  menuItem MenuItemModificationGroup[]
+  menuItem      MenuItemModificationGroup[]
   modifications Modification[]
 }
 
 model Modification {
-  id String @id @default(dbgenerated("uuid_generate_v4()")) @db.Uuid
-  modificationGroupId String @db.Uuid
-  name String
-  price Decimal @db.Decimal(12,2) @default(0)
-  createdAt DateTime @default(now())
-  updatedAt DateTime @default(now())
+  id                  String   @id @default(dbgenerated("uuid_generate_v4()")) @db.Uuid
+  modificationGroupId String   @db.Uuid
+  name                String
+  price               Decimal  @default(0) @db.Decimal(12, 2)
+  createdAt           DateTime @default(now())
+  updatedAt           DateTime @default(now())
 
   modificationGroup ModificationGroup @relation(fields: [modificationGroupId], references: [id])
 
   @@unique([modificationGroupId, name])
 }
 
-model Order{
-  id String @id @default(dbgenerated("uuid_generate_v4()")) @db.Uuid
-  customerId String @db.Uuid 
-  restaurantId String @db.Uuid
-  orderDate DateTime @default(now())
-  status String
-  totalAmount Int
-  freeDelivery Boolean    @default(false)
-  address String
-  createdAt DateTime @default(now())
-  updatedAt DateTime @default(now())
+model Order {
+  id           String   @id @default(dbgenerated("uuid_generate_v4()")) @db.Uuid
+  customerId   String   @db.Uuid
+  restaurantId String   @db.Uuid
+  orderDate    DateTime @default(now())
+  status       String
+  totalAmount  Int
+  freeDelivery Boolean  @default(false)
+  address      String
+  createdAt    DateTime @default(now())
+  updatedAt    DateTime @default(now())
 
   orderLines OrderLine[]
 
   restaurant Restaurant @relation(fields: [id], references: [orderId])
-  customer Customer @relation(fields: [customerId],references: [id])
+  customer   Customer   @relation(fields: [customerId], references: [id])
 }
 
-model OrderLine{
-  id String @id @default(dbgenerated("uuid_generate_v4()")) @db.Uuid
-  menuItemId String @db.Uuid
-  orderId String @db.Uuid 
-  quantity Int
+model OrderLine {
+  id         String   @id @default(dbgenerated("uuid_generate_v4()")) @db.Uuid
+  menuItemId String   @db.Uuid
+  orderId    String   @db.Uuid
+  quantity   Int
   totalPrice Int
-  createdAt DateTime @default(now())
-  updatedAt DateTime @default(now())
-  
-  menuItem MenuItem @relation(fields: [menuItemId],references: [id])
-  order Order @relation(fields: [orderId],references: [id])
-}
\ No newline at end of file
+  createdAt  DateTime @default(now())
+  updatedAt  DateTime @default(now())
+
+  menuItem MenuItem @relation(fields: [menuItemId], references: [id])
+  order    Order    @relation(fields: [orderId], references: [id])
+}
+
+model Supplier {
+  id         String       @id @default(dbgenerated("uuid_generate_v4()")) @db.Uuid
+  name       String
+  contacts   String
+  bin        String
+  Ingredient Ingredient[]
+}
+
+enum Unit {
```

Comment:
let's remove db level enum, because enums are hard to work with in db. you can add enum, but cannot easily delete it. Also, it is easier to add and manage enums in code level